NEWS.rst

```diff
@@ -1,4 +1,4 @@
-Version 0.10.0, 2018-10-07
+Version 0.10.1, 2018-10-08
```

Comment:
Erm, weren't we going to simply overwrite `0.10.0`?

Reply:
It's been tagged already. Rewriting published tags is messy business.